examples/JiraProjectManagement/Ballerina.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 org = "ansak"
-name = "create_new_project"
+name = "JiraProjectManagement"
```

Comment:
As in this comment, the name has to be in the format `jira-project-and-issue-management`.
https://github.com/ballerina-platform/module-ballerinax-jira/pull/5#issuecomment-2484584030

Reply:
done